### PR TITLE
@thunderstore/sertra: fix Prettier/ESLint linting conflict

### PR DESCRIPTION
**Pull request by anttimaki**

### CODE REVIEW

**Comment by anttimaki** [comment]
Current dependencies on/for this PR:
* master
  * **PR #517** <a href="https://app.graphite.dev/github/pr/thunderstore-io/thunderstore-ui/517" target="_blank">Graphite</a>  👈
    * **PR #518** <a href="https://app.graphite.dev/github/pr/thunderstore-io/thunderstore-ui/518" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/thunderstore-io/thunderstore-ui/517?utm_source=stack-comment).

**Review by Oksamies — APPROVED** [review]

**Review by MythicManiac — APPROVED** [review]

**Comment by MythicManiac** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/thunderstore-io/thunderstore-ui/517?utm_source=stack-comment).


Job Reference: 7K1Z41xmMJoAQsKTIbn5

**Pull request merged**
